study-builder/studies/prion/consent.conf

```diff
@@ -29,6 +29,7 @@
       "text": ${i18n.zh.consentName}
     }
   ],
+  "translatedSecondNames": [],
```

Comment:
I recommend adding an `include` at the top to pull in the defaults, so these files are less likely to break. Something like:
```
{
  include required("../snippets/activity-general-form.conf"),
  "studyGuid": "PRION",
  ... the rest of the stuff ...
}
```